apps/saleor-app-checkout/backend/payments/providers/adyen/webhookHandler.ts

```diff
@@ -57,10 +61,18 @@ export const getOrderId = async (
     // it's possible to get notification metadata directly from notification itself (undocumented)
     console.warn("checkout.getPaymentLinks failed");
```

Comment:
I think we can move this `console.warn` into `getOrderIdFromNotificationItselfUndocumented`

Reply:
We can't because that function is used in multiple places, unrelated to the error thrown here.